File: frontend/src/app/models/common/buildDto.ts
/* Auto Generated */

import { ProjectDTO } from "./projectDTO"
import { UserDTO } from "./../User/userDTO"

export interface BuildDto {
    id: number;
    buildMessage: string;
    buildStarted: Date;
    buildFinished?: Date;
    buildStatus: any;
    project: ProjectDTO;
    user: UserDTO;
}

File: backend/IDE.Common/DTO/Common/BuildDto.cs
using IDE.Common.DTO.User;
using IDE.Common.Enums;
using System;

namespace IDE.Common.DTO.Common
{
    public class BuildDTO
    {
        public string BuildMessage { get; set; }
        public DateTime BuildStarted { get; set; }
        public DateTime? BuildFinished { get; set; }
        public BuildStatus BuildStatus { get; set; }
        public ProjectDTO Project { get; set; }
        public UserDTO User { get; set; }
    }
}
